src/app/static/src/app/components/UserHeader.vue

```diff
@@ -23,6 +23,7 @@
                         <input type="file" style="display: none;" ref="loadFile" v-on:change="load" accept=".json">
                     </div>
                 </div>
+                <a href="https://forms.gle/QxCT1b4ScLqKPg6a7" target="_blank" style="margin-right: 10px">Report a bug</a>
```

Comment:
Bootstrap has a bunch of extremely useful spacing utilities that can be used here instead of the inline style. And how about putting a right border also so there's a clear separation between this and the logout link? So the classes would be "pr-2 mr-2 border-right"

For reference: https://getbootstrap.com/docs/4.4/utilities/spacing/